src/ui/display_write_video_thread.py

```diff
@@ -25,10 +26,11 @@ def run(self):
 
         self.model = Model(weights_path)
         self.Fourcc = cv2.VideoWriter_fourcc(*'mp4v')
-        self.Output = cv2.VideoWriter('video_recording.mp4', self.Fourcc, 20, (640, 480))
+        self.Output = cv2.VideoWriter('video_recording'+str(self.cameraNumber)+'.mp4', self.Fourcc, 20, (640, 480))
+
         #self.Capture = cv2.VideoCapture(0)
 
-        while self.ThreadActive: 
+        while True: 
```

Comment:
This causes a crash on exit because we don't have an exit condition for this loop. 
This was changed from main where it's self.ThreadActive which solved this, is there a reason why?

Reply:
Sorry, I realized I might not have pushed this change on main. The comment stands but ignore the 'why was this changed' bit.

Reply:
That's my bad, I wanted my original code after trying out the Video() class so I copied and pasted my code from the old branch. Changing now.